cmd/kubeadm/app/cmd/certs.go

```diff
@@ -390,73 +399,59 @@ func newCmdCertsExpiration(out io.Writer, kdir string) *cobra.Command {
 				return err
 			}
 
-			// Get all the certificate expiration info
-			yesNo := func(b bool) string {
-				if b {
-					return "yes"
-				}
-				return "no"
-			}
-			w := tabwriter.NewWriter(out, 10, 4, 3, ' ', 0)
-			fmt.Fprintln(w, "CERTIFICATE\tEXPIRES\tRESIDUAL TIME\tCERTIFICATE AUTHORITY\tEXTERNALLY MANAGED")
+			info := &outputapiv1alpha3.ExpirationInfo{}
```

Comment:
can we move the whole code in a new helper function and add a unit test for it?
no need to be very verbose in the unit test, just test a basic JSON output scenario with some certs and CAs, maybe some missing certs too.

Reply:
@neolit123 added a unit test for this command. please see https://github.com/kubernetes/kubernetes/compare/0b3e5411c83eef6f658d037eb4c5e0a3847df0fc..6ec538fc25c17a8c72855c8117865be9f1fb3524